.prettierrc

```
@@ -0,0 +1,10 @@
+{
+    "semi": true,
+    "singleQuote": true,
+    "trailingComma": "es5",
+    "printWidth": 80,
+    "tabWidth": 2,
+    "useTabs": false,
+    "bracketSpacing": true,
+    "arrowParens": "always"
+}
```

Recomenda-se deixar uma linha em branco ao final de cada arquivo (independente da extensão).

Ao deixar tal linha em branco, ferramentas como o GitHub não exibem o ícone de `No newline at end of file`.

A propósito, é possível configurar o ESLint para garantir que tal regra seja sempre cumprida.

```suggestion
}

```